### PR TITLE
procs: add page

### PR DESCRIPTION
**Pull request by navneetankur**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/procs.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# procs
+
+> procs displays information about the active processes.
+> It can be a used as ps or top alternative
+> More information: <https://github.com/dalance/procs>
```
Comment (marchersimon) [inline pages/linux/procs.md:5]:
```suggestion
> It can be a used as ps or top alternative.
> More information: <https://github.com/dalance/procs>.
```

File: pages/linux/procs.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# procs
+
+> procs displays information about the active processes.
```
Comment (marchersimon) [inline pages/linux/procs.md:3]:
```suggestion
> Displays information about the active processes.
```
This is what we typically use

File: pages/linux/procs.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+> It can be a used as ps or top alternative
+> More information: <https://github.com/dalance/procs>
+
+- Show information about all the processes:
```
Comment (marchersimon) [inline pages/linux/procs.md:7]: Could you maybe elaborate a little more what it shows?

File: pages/linux/procs.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+`procs`
+
+- Show information about process with command or user containg `zsh`:
```
Comment (marchersimon) [inline pages/linux/procs.md:11]: I'd rephrase this a little
```suggestion
- Show information about processes containing `zsh` and the commands which started them:
```

File: pages/linux/procs.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+`procs {{zsh}}`
+
+- Show information about all processes sorted in [d]escending order by `cpu` time:
+
+`procs --sortd cpu`
```
Comment (marchersimon) [inline pages/linux/procs.md:17]:
```suggestion
- Show information about all processes sorted by CPU time in [d]escending or [a]scending order:

`procs --sort{{d}}|--sort{{a}} cpu`
```
I hope that's correct

File: pages/linux/procs.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+- Show information about processes with pid `41` or (command or user) containing (`zsh` `or` `firefox`):
+
+`procs --or {{zsh}} {{41}} {{firefox}}`
```
Comment (marchersimon) [inline pages/linux/procs.md:21]: I'm a little confused about this. So `--or` includes all three of them right?

File: pages/linux/procs.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+- Show information about processes with pid `41` and (command or user) containing `zsh`:
+
+`procs --and {{41}} {{zsh}}`
```
Comment (marchersimon) [inline pages/linux/procs.md:25]: How does the tool know that I don't want to search for a process containing "41"?

**Review by marchersimon — COMMENTED** [review]
Thanks for you ccontributions. I left a few suggestions

File: pages/linux/procs.md (hunk 6 above)
Comment (navneetankur) [reply pages/linux/procs.md:21]: Yes.

File: pages/linux/procs.md (hunk 7 above)
Comment (navneetankur) [reply pages/linux/procs.md:25]: Numeric values are matched to pid.

File: pages/linux/procs.md (hunk 4 above)
Comment (navneetankur) [reply pages/linux/procs.md:11]: How about:
Show information about processes, if the commands which started them contain `zsh` :

Comment (marchersimon) [reply pages/linux/procs.md:11]: I'm not sure if `if` is the right word here. Let's wait for some other opinions

File: pages/linux/procs.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+# procs
+
+> Displays information about the active processes.
+> It can be a used as ps or top alternative.
+> More information: <https://github.com/dalance/procs>.
+
+- Show information like pid, user, cpu usage, memory usage, command which started them about all the processes:
```
Comment (marchersimon) [inline pages/linux/procs.md:7]: I think this is a little easier to read:
```suggestion
- List all processes with PID, user, CPU usage, memory usage and the command which started them:
```

File: pages/linux/procs.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+
+- Show information about process with command or user containg `zsh`:
+
+`procs {{zsh}}`
+
+- Show information about all processes sorted by CPU time in [d]escending or [a]scending order:
+
+`procs --sort{{d}}|--sort{{a}} cpu`
+
+- Show information about processes with pid `41` or (command or user) containing (`zsh` `or` `firefox`):
```
Comment (marchersimon) [inline pages/linux/procs.md:19]:
```suggestion
- Show information about processes with PID `41` or (command or user) containing (`zsh` `or` `firefox`):
```

File: pages/linux/procs.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+
+`procs --or {{zsh}} {{41}} {{firefox}}`
+
+- Show information about processes with pid `41` and (command or user) containing `zsh`:
```
Comment (marchersimon) [inline pages/linux/procs.md:23]:
```suggestion
- Show information about processes with PID `41` and (command or user) containing `zsh`:
```

File: pages/linux/procs.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
+# procs
+
+> Displays information about the active processes.
+> It can be a used as ps or top alternative.
+> More information: <https://github.com/dalance/procs>.
+
+- List all processes with PID, user, CPU usage, memory usage and the command which started them:
```
Comment (CleanMachine1) [inline pages/linux/procs.md:7]:
```suggestion
- List all processes showing the: PID, user, CPU usage, memory usage and, the command which started them:
```

File: pages/linux/procs.md [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+
+`procs`
+
+- Show information about process with command or user containg `zsh`:
+
+`procs {{zsh}}`
+
+- Show information about all processes sorted by CPU time in [d]escending or [a]scending order:
+
+`procs --sort{{d}}|--sort{{a}} cpu`
```
Comment (CleanMachine1) [inline pages/linux/procs.md:17]:
```suggestion
`procs --sort{{a|d}} cpu`
```

File: pages/linux/procs.md [hunk 13]
```diff
@@ -0,0 +1,25 @@
+# procs
+
+> Displays information about the active processes.
+> It can be a used as ps or top alternative.
```
Comment (CleanMachine1) [inline pages/linux/procs.md:4]:
```suggestion
```
Needless

File: pages/linux/procs.md [hunk 14]
```diff
@@ -0,0 +1,25 @@
+# procs
+
+> Displays information about the active processes.
```
Comment (CleanMachine1) [inline pages/linux/procs.md:3]:
```suggestion
> Display information about the active processes.
```

Doesn't need to be plural

File: pages/linux/procs.md [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+
+- List all processes with PID, user, CPU usage, memory usage and the command which started them:
+
+`procs`
+
+- Show information about process with command or user containg `zsh`:
+
+`procs {{zsh}}`
+
+- Show information about all processes sorted by CPU time in [d]escending or [a]scending order:
```
Comment (CleanMachine1) [inline pages/linux/procs.md:15]: What is CPU time, if that is time since it started the process then change it to 

```suggestion
- Show information about all processes sorted by running time in an [a]scending or [d]escending order:
```

Comment (CleanMachine1) [reply pages/linux/procs.md:15]: Also the orders of a and d should be switched.

File: pages/linux/procs.md [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+
+`procs --sort{{d}}|--sort{{a}} cpu`
+
+- Show information about processes with PID `41` or (command or user) containing (`zsh` `or` `firefox`):
```
Comment (CleanMachine1) [inline pages/linux/procs.md:19]:
```suggestion
- Show information about processes with either a PID, command or, user containing (`zsh` `or` `firefox`):
```

File: pages/linux/procs.md [hunk 17]
```diff
@@ -0,0 +1,25 @@
...
+
+`procs --or {{zsh}} {{41}} {{firefox}}`
```
Comment (CleanMachine1) [inline pages/linux/procs.md:21]:
```suggestion
`procs --or {{zsh}} {{PID|command|user}} {{firefox}}`
```

File: pages/linux/procs.md (hunk 7 above)
Comment (CleanMachine1) [reply pages/linux/procs.md:25]:
```suggestion
`procs --and {{PID}} {{zsh}}`
```

File: pages/linux/procs.md [hunk 18]
```diff
@@ -0,0 +1,25 @@
...
+
+- Show information about processes with PID `41` and (command or user) containing `zsh`:
```
Comment (CleanMachine1) [inline pages/linux/procs.md:23]:
```suggestion
- Show information about processes with either a PID, command or, user containing `zsh`:
```

**Review by CleanMachine1 — COMMENTED** [review]
This is a start :smile: 

Everyone makes errors on their first PR, especially our awkward syntax :)

File: pages/linux/procs.md (hunk 11 above)
Comment (marchersimon) [reply pages/linux/procs.md:7]: I don't agree with the `:` and the comma should go before the `and`:
```suggestion
- List all processes showing the PID, user, CPU usage, memory usage, and the command which started them:
```

File: pages/linux/procs.md (hunk 12 above)
Comment (marchersimon) [reply pages/linux/procs.md:17]: That syntax was originally my suggestion. Both versions are weird, but when comparing the output I definitively prefer the way it's now:
image

File: pages/linux/procs.md (hunk 13 above)
Comment (marchersimon) [reply pages/linux/procs.md:4]: Not directly needless, but also not really important.

File: pages/linux/procs.md (hunk 15 above)
Comment (marchersimon) [reply pages/linux/procs.md:15]: CPU time is correct. I don't know what it is exactly, but it's definitively a widely used term.
I agree with swapping ascending and descending, but nothing else:
```suggestion
- Show information about all processes sorted by CPU time in [a]scending or [d]escending order:
```

File: pages/linux/procs.md (hunk 16 above)
Comment (marchersimon) [reply pages/linux/procs.md:19]: The comma should go before the `or`.

File: pages/linux/procs.md (hunk 17 above)
Comment (marchersimon) [reply pages/linux/procs.md:21]: Not a bad idea, but I'd put it at the end and maybe keep the PID example:
```suggestion
`procs --or {{PID|command|user}} {{41}} {{firefox}}`
```

File: pages/linux/procs.md (hunk 18 above)
Comment (marchersimon) [reply pages/linux/procs.md:23]: `either` can only be used with `or` and we have `and` here, so it should be `both`. Also, we can't use the Oxford comma here, since it's not part of the enumeration.
```suggestion
- Show information about processes with both PID `41` and a command or user containing `zsh`:
```

File: pages/linux/procs.md (hunk 16 above)
Comment (CleanMachine1) [reply pages/linux/procs.md:19]: :scream: whoops

File: pages/linux/procs.md (hunk 12 above)
Comment (navarroaxel) [reply pages/linux/procs.md:17]: The `a` and `d` are part of the argument name, so I think the correct is:
```suggestion
`procs {{--sortd|--sorta}} cpu`
```

File: pages/linux/procs.md (hunk 16 above)
Comment (navarroaxel) [reply pages/linux/procs.md:19]: Why the second `or` has back ticks?

Comment (marchersimon) [reply pages/linux/procs.md:19]: Good question. I don't think we need them here.

File: pages/linux/procs.md (hunk 4 above)
Comment (marchersimon) [reply pages/linux/procs.md:10]: Actually, why not?
```suggestion
- Show information about processes, if the commands which started them contain `zsh`:
```

**Review by marchersimon — APPROVED** [review]
Thanks @navneetankur, sorry this took so long.

**Comment by navneetankur** [comment]
> Thanks @navneetankur, sorry this took so long.

No problem. and thank you for making this page understandable by everyone instead of just me. :)

**Review by CleanMachine1 — APPROVED** [review]
